### ISSUE TITLE
Connection reset after 14 steps

### ISSUE THREAD

**Issue by rtens** [zells/seven#15]
The connection to uberspace is reset after 14 steps are taken with the turtle

**Comment by rtens** [comment]
Turns out the server crashes

```
@400000005a0ea02a331e20d4 /home/rtens/zells-seven/javascript/node_modules/msgpack-lite/lib/read-format.js:63
@400000005a0ea02a331e2c8c   var k = new Array(len);
@400000005a0ea02a331e2c8c                          
@400000005a0ea02a331e3074 
@400000005a0ea02a331e3074 Error: BUFFER_SHORTAGE
@400000005a0ea02a331e345c     at DecodeBuffer.reserve (/home/rtens/zells-seven/javascript/node_modules/msgpack-lite/lib/flex-buffer.js:64:41)
@400000005a0ea02a331e3844     at /home/rtens/zells-seven/javascript/node_modules/msgpack-lite/lib/read-format.js:154:25
@400000005a0ea02a331f556c     at decode (/home/rtens/zells-seven/javascript/node_modules/msgpack-lite/lib/read-core.js:25:12)
@400000005a0ea02a331f5d3c     at map_to_obj (/home/rtens/zells-seven/javascript/node_modules/msgpack-lite/lib/read-format.js:52:12)
@400000005a0ea02a331f6124     at /home/rtens/zells-seven/javascript/node_modules/msgpack-lite/lib/read-token.js:159:12
@400000005a0ea02a331f650c     at decode (/home/rtens/zells-seven/javascript/node_modules/msgpack-lite/lib/read-core.js:25:12)
@400000005a0ea02a331fda3c     at map_to_obj (/home/rtens/zells-seven/javascript/node_modules/msgpack-lite/lib/read-format.js:52:12)
@400000005a0ea02a331fde24     at /home/rtens/zells-seven/javascript/node_modules/msgpack-lite/lib/read-token.js:159:12
@400000005a0ea02a331fe5f4     at decode (/home/rtens/zells-seven/javascript/node_modules/msgpack-lite/lib/read-core.js:25:12)
@400000005a0ea02a33203be4     at map_to_obj (/home/rtens/zells-seven/javascript/node_modules/msgpack-lite/lib/read-format.js:52:12)
```

**Comment by rtens** [comment]
Maybe use other msgpack library

https://www.npmjs.com/browse/keyword/msgpack

Also see https://github.com/darrachequesne/notepack

**Comment by rtens** [comment]
Solved by 5e99ed3474e9b9d09f14def8b28e34365917daa6
